### PR TITLE
Fix the ci

### PR DESCRIPTION
**Pull request by jockos**
| Q             | A
| ------------- | ---
| Bug fix?      | no 
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

Just fix the CI tests

### CODE REVIEW

**Comment by alanpoulain** [comment]
Related: https://github.com/api-platform/core/pull/3923

File: src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php [hunk 1]
```diff
@@ -256,7 +256,7 @@ private function getGraphQlSubscriptionUpdates($object, array $options, string $
     }
 
     /**
-     * @var string|string[] $iri
+     * @var string|string[]
```
Comment (alanpoulain) [inline src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php:259]: Should be a param.

File: src/Swagger/Serializer/DocumentationNormalizer.php [hunk 2]
```diff
@@ -697,13 +697,13 @@ private function computeDoc(bool $v3, Documentation $documentation, \ArrayObject
             $security[] = [$key => []];
         }
 
-        if ($v3) {
-            if ($securityDefinitions && $security) {
+        if ($securityDefinitions && !empty($security)) {
```
Comment (alanpoulain) [inline src/Swagger/Serializer/DocumentationNormalizer.php:700]: We avoid to use `empty` if possible.

File: src/Swagger/Serializer/DocumentationNormalizer.php [hunk 3]
```diff
@@ -697,13 +697,13 @@ private function computeDoc(bool $v3, Documentation $documentation, \ArrayObject
...
+            if ($v3) {
+                $docs['security'] = $security;
```
Comment (alanpoulain) [inline src/Swagger/Serializer/DocumentationNormalizer.php:702]: This line should be outside the condition.

File: features/doctrine/search_filter.feature [hunk 4]
```diff
@@ -624,6 +624,7 @@ Feature: Search filter on collections
     Given there are 30 dummy objects
     When I send a "GET" request to "/dummies?id=9.99"
     Then the response status code should be 200
+    Then print last JSON response
```
Comment (dunglas) [inline features/doctrine/search_filter.feature:627]: To remove

Comment (soyuka) [inline features/doctrine/search_filter.feature:627]:
```suggestion
```

**Comment by soyuka** [comment]
Thanks @jockos !

**Pull request merged**